formstack-baton-requests/src/main/scala/com/gu/identity/formstackbatonrequests/FormstackSarHandler.scala

```diff
@@ -16,7 +15,8 @@ case class FormstackSarHandler(s3Client: S3Client, lambdaClient: LambdaClient, s
 
     val performSarRequest = SarPerformRequest(
       initiationReference,
-      request.subjectEmail
+      request.subjectEmail,
+      "formstack"
```

Comment:
why do we need this?

Reply:
I've added a check (that will appear in my next PR) that just checks this field is present for perform requests and init requests